app/validators/collection_druid_validator.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+# Validates that collection exists.
+class CollectionDruidValidator < ActiveModel::EachValidator
+  def validate_each(record, attribute, value)
+    return unless Settings.validate_collection_druid
+    return if record.errors[attribute].present?
```

Comment:
If there are already errors for this attr, don't add a collection druid error to the list?

Reply:
Yes, though the motivation is to stop an invalid druid from causing an OpenAPI error when calling find. I added a comment to reflect this.